plugins/standard_tests/newhelm/tests/configs/safe-cae-benign.toml

```diff
@@ -0,0 +1,3 @@
+uid = "mls_ais_wg_safe_cae_benign"
+version = "data_08mar2024"
```

Comment:
prefer [ISO format](https://xkcd.com/1179/) so that lexicographical order is also chronological order - `either 20240308` or `2024-03-08`

Reply:
Here the version is just referring to the (locked) google drive folder that this version of SafeTest data is in